core/src/pretty.rs

```diff
@@ -833,9 +828,7 @@ where
                     .append(typ.pretty(allocator));
 
                 let builder = if let RecordRowsF::Extend { .. } = tail.0 {
-                    builder
-                        .append(allocator.text(","))
-                        .append(allocator.space())
+                    builder.append(allocator.text(",")).append(allocator.line())
```

Comment:
What is the difference between using `space()` and `line()`? I thought the pretty printer would automatically insert line breaks when it needs to

Reply:
When a particular `group()` doesn't fit on a single line, the pretty printer will first turn *every* `line()` into a line break and after that try to see if any more line breaks are needed, for instance at `space()`. So, this change is the difference between getting
```
{
  a, b, c,
  d, e,
}
```
and
```
{
  a,
  b,
  c,
  d,
  e,
}
```

It seems that's our formatting preference (also in topiary): if there are already multiple lines required for a record, then each record field should get its own.